# Tag 10

In [1]:
from math import pi, sin, cos, tan, asin, acos, atan, sqrt, degrees as deg, radians as rad

Sinussatz:

$\frac{a}{\sin(\alpha)} = \frac{b}{\sin(\beta)} = \frac{c}{\sin(\gamma)}$

bzw. Nenner und Zähler vertauscht.

Cosinussatz:

$c^2 = a^2+b^2-2ab\cos(\gamma)$

bzw.
$a^2 = b^2+c^2-2ab\cos(\alpha)$, 
$b^2 = a^2+c^2-2ab\cos(\beta)$

https://www.wolframalpha.com/input?i=solve+a%5E2+%3D+b%5E2%2Bc%5E2+-2*a*b*cos%28alpha%29+for+b

## 41

### a)

$b = 161m, c=117m, \gamma=28,1°$

Zwei Seiten und gegenüberliegenden Winkel -> Sinussatz

$\frac{\sin(\beta)}{b} = \frac{\sin(\gamma)}{c} | *b$

$\Leftrightarrow \sin(\beta) = \frac{b\sin(\gamma)}{c} | \sin^{-1}(.)$

$\Leftrightarrow \beta = \sin^{-1}(\frac{b\sin(\gamma)}{c})$

In [2]:
b = 161
c = 117
gamma = 28.1

beta = deg(asin(b*sin(rad(gamma))/c))
beta

40.40185487354103

In [3]:
alpha = 180 - beta - gamma
alpha

111.49814512645898

Sinussatz für a

$\frac{a}{\sin(\alpha)} = \frac{b}{\sin(\beta)} | * \sin(\alpha)$

$\Leftrightarrow a = \frac{b\sin(\alpha)}{\sin(\beta)}$

In [4]:
a = b * sin(rad(alpha)) / sin(rad(beta))
a

231.11995246911482

$\beta \approx 40.40°, \alpha \approx 111.50°, a \approx 231.12m$

### b)

$a=301mm, b=402mm, c=439mm$

Drei Seiten -> Kosinussatz

$c^2 = a^2+b^2-2ab\cos(\gamma) | - a^2 - b^2$

$\Leftrightarrow c^2 -a^2 - b^2 = -2ab\cos(\gamma) | / (-2ab)$

$\Leftrightarrow \frac{c^2 -a^2 - b^2}{-2ab} = \cos(\gamma) | \arccos(.)$

$\Leftrightarrow \arccos(\frac{c^2 -a^2 - b^2}{-2ab}) = \gamma$

In [5]:
a = 301
b = 402
c = 439

gamma = deg(acos((c**2 - a**2 - b**2)/(-2*a*b)))
gamma

75.77102628576468

In [6]:
beta = deg(acos((b**2 - a**2 - c**2)/(-2*a*c)))
beta

62.576231955043994

In [7]:
alpha = deg(acos((a**2 - b**2 - c**2)/(-2*b*c)))
alpha

41.65274175919133

$\alpha \approx 41.65°, \beta \approx 62,58°, \gamma \approx 75,77°$

## 42

Höhe eines gleichseitigen Dreiecks mit Kantenlänge $1$.

$1^2 = h^2 + (\frac{1}{2})^2 | - (\frac{1}{2})^2$

$\Leftrightarrow 1 - \frac{1}{4} = h^2 | \sqrt{.}$

$\Leftrightarrow \sqrt{\frac{3}{4}} = h$

$\Leftrightarrow \frac{\sqrt{3}}{\sqrt{4}} = h$

$\Leftrightarrow \frac{\sqrt{3}}{2} = h$

$\sin(60°) = \frac{\frac{\sqrt{3}}{2}}{1} = \frac{\sqrt{3}}{2}$

$\cos(60°) = \frac{\frac{1}{2}}{1} = \frac{1}{2}$

$\tan(60°) = \frac{\sin(60°)}{\cos(60°)} = \frac{\frac{\sqrt{3}}{2}}{\frac{1}{2}} = \sqrt{3}$

$\cot(60°) = \frac{1}{\tan(60°)} = \frac{1}{\sqrt{3}}$

## 43

Schnittfläche aus einer Kante und der Raumdiagonale. Dies ist ein Rechteck mit kantenlänge 1 und $\sqrt{2}$.

In [8]:
deg(atan(sqrt(2)))

54.735610317245346

## 44

Winkel zwischen den beiden verschieden langen Seiten eines gleichschenkligen Dreiecks. Die eine Seite hat die Länge 1, die anderen die Höhe des Dreiecks. Aus Aufgabe 42 ist die Höhe bekannt. Nun der Kosinussatz:

$\cos^{-1}(\frac{(\frac{\sqrt{3}}{2})^2 -(\frac{\sqrt{3}}{2})^2 - 1}{-2*\frac{\sqrt{3}}{2}*1})$

$= \cos^{-1}(\frac{ - 1}{-2*\frac{\sqrt{3}}{2}*1})$

$= \cos^{-1}(\frac{1}{\sqrt{3}})$

In [9]:
deg(acos(1/sqrt(3)))

54.735610317245346

## 45

Gleichschenkliges Dreieck mit $10 cm$ länge der gleichlangen Seite. Der eingeschlossene Winkel ist $\frac{360°}{8}$. Kosinussatz:

$r = \sqrt{(10 cm)^2 + (10 cm)^2 - 2*(10 cm)*(10 cm)*\cos(\frac{360°}{8})}$

In [10]:
sqrt(10**2 + 10**2 - 2 * 10 * 10 * cos(rad(360/8)))

7.653668647301795

## 46

Ein Dreieck mit Seitenlänge eins und die anderen beiden haben die Länge der halben Raumdiagonalen.

Die Länge der Raumdiagonale wird mit zwei rechtwinkligen Dreiecken bestimmt.

$\sqrt{1^2 + 1^2}$ für die Diagonale einer Fläche. Die Raumdiagonale ist die Kante des Dreiecks durch diese Diagonale und einer Kante.

$\sqrt{(\sqrt{1^2 + 1^2})^2 + 1^2}$

$= \sqrt{1^2 + 1^2 + 1^2}$

$= \sqrt{3}$

Kosinussatz für den Winkel:

$\cos^{-1}(\frac{1^2 -(\frac{\sqrt{3}}{2})^2 - (\frac{\sqrt{3}}{2})^2}{-2*\sqrt{3}*\sqrt{3}})$

$= \cos^{-1}(\frac{1^2 -(\sqrt{3})^2 - (\sqrt{3})^2}{-2*\frac{\sqrt{3}}{2}*\frac{\sqrt{3}}{2}})$

$= \cos^{-1}(\frac{1 - \frac{3}{4} - \frac{3}{4}}{-\frac{3}{2}})$

$= \cos^{-1}(\frac{- \frac{2}{4}}{-\frac{3}{2}})$

$= \cos^{-1}(\frac{\frac{2}{4}}{\frac{3}{2}})$

$= \cos^{-1}(\frac{2}{4}*\frac{2}{3})$

$= \cos^{-1}(\frac{1}{3})$

In [11]:
deg(acos(1/3))

70.52877936550931

In [12]:
180 - 2* deg(atan(sqrt(2)))

70.52877936550931

## 47

Zuerst b)

### b)

$w = 2\pi * f = 2\pi * 1450 \frac{1}{1 min} * \frac{1 min}{60 s}$

In [13]:
2 * pi * 1450 / 60

151.84364492350667

Die Winkelgeschwindigkeit ist $w \approx 151.84364492350667\frac{1}{s}$

### a)

Der Radius ist $\frac{2 mm}{2} = 1 mm$

$w*r = 2\pi * f = 2\pi * 1450 \frac{1}{1 min} * \frac{1 min}{60 s} * 1 mm * \frac{1 m }{10^3 mm}$

In [14]:
2 * pi * 1450 / 60 * 1 / 10**3

0.15184364492350666

Die Umfangsgeschwindigkeit ist $0.15184364492350666\frac{m}{s}$

## 48

### a)

$\cos(58°) = \frac{\frac{l}{2}}{\frac{8,35 ms * 340 \frac{m}{s}}{2}}$

$\Leftrightarrow \cos(58°) = \frac{l}{8,35 ms * 340 \frac{m}{s}*\frac{1 s}{1000 ms}}$

$\Leftrightarrow \cos(58°) = \frac{l}{2,839m} | * 2.839 m$

$\Leftrightarrow \cos(58°) * 2.839 m = l$

In [15]:
8.35 * 0.34 * cos(rad(58))

1.5044407911580686

Der Abstand zwischen Sender und Empfänger beträgt 1,5 m.

### b)

Genau genommen nicht der Abstand. Der wäre 1.4195 m, denn die Gesammtstrecke wurde für a) schon ausgerechnet und muss nur halbiert werden.

$1.4195^2 = (\frac{l}{2})^2 + a^2$

$1.4195^2 - (\frac{l}{2})^2=  a^2 |\sqrt{.}$

$\sqrt{1.4195^2 - (\frac{\cos(58°) * 2.839 m}{2})^2}=  a$


In [16]:
sqrt((8.35 * 0.34 / 2)**2 - (8.35 * 0.34 * cos(rad(58)) / 2)**2)

1.2038042724940468

## 49

Es gilt:

$\sin(\alpha\pm\beta) = \sin(\alpha)\cos(\beta)\pm\cos(\alpha)sin(\beta)$

$\cos(\alpha\pm\beta) = \cos(\alpha)\cos(\beta)\mp\sin(\alpha)sin(\beta)$

$\sin(x)^2 + \cos(x)^2 = 1$

### a)

$\sin(2\alpha) = \sin(\alpha+\alpha)$

$ = \sin(\alpha)\cos(\alpha)+\cos(\alpha)sin(\alpha)$

$ = 2\sin(\alpha)\cos(\alpha)$

### b)

$\cos(2\alpha) = \cos(\alpha+\alpha)$

$ = \cos(\alpha)\cos(\alpha)-\sin(\alpha)sin(\alpha)$

$ = \cos(\alpha)^2-\sin(\alpha)^2$

Ersetzen $\sin(x)^2 + \cos(x)^2 = 1 \Leftrightarrow \sin(x)^2 = 1 - \cos(x)^2$

$ = \cos(\alpha)^2 - (1 - \cos(x)^2)$

$ = 2\cos(\alpha)^2 - 1$